docs/modules/ROOT/pages/how-to/customize-cypher.adoc

```diff
@@ -47,8 +47,8 @@ Generates the following Cypher:
 
 [source, cypher]
 ----
-MATCH(this0:Movie)
-MATCH(this0:Person)
+MATCH(:Movie)
+MATCH(:Person)
```

Comment:
You passed the Node in the Pattern above so I believe here it will be printed as `this0:Movie` and `this1:Person`